Dockerfile

```diff
@@ -1,17 +1,19 @@
-FROM python:3.12-rc-alpine
+FROM python:3.13.0rc2-alpine3.19
```

Comment:
This project doesn't officially support Python 3.13 yet, and I would like to roll it once Python 3.13 is officially released.